app/assets/src/components/Samples.jsx

```diff
@@ -2180,4 +2328,34 @@ function AddUserModalMemberArea({ state, parent }) {
   );
 }
 
+function ActiveInactiveButton({
+  label,
+  onClick,
+  enabled,
+  icon,
+  outerClass,
+  enabledClass,
+  disabledClass
+}) {
+  let cohort_button_inner = (
```

Comment:
Can you just remove the `cohort_` prefix?

Reply:
ah yes, thanks